OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -109,6 +109,15 @@ public interface IDisableMove { bool MoveDisabled(Actor self); }
 
 	public interface ISeedableResource { void Seed(Actor self); }
 
+	public interface IVoiced
+	{
+		bool PlayVoice(string phrase, Actor voicedActor, string variant);
+		bool PlayVoiceLocal(string phrase, Actor voicedActor, string variant, WPos pos, float volume);
+		bool HasVoices(Actor actor);
+		bool HasVoice(Actor actor, string voice);
+		string VoiceSet { get; }
```

Comment:
Please move this property before the method declarations (conventions state properties go before methods).